modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm

```diff
@@ -167,9 +167,12 @@ sub resource_classes {
     my ($self) = @_;
     return {
         %{$self->SUPER::resource_classes},
-        'default' => { 'LSF' => '-q production -R"select[mem>1500] rusage[mem=1500]" -M1500'},
-        'medium'  => { 'LSF' => '-q production -R"select[mem>4500] rusage[mem=4500]" -M4500'},
-        'high'    => { 'LSF' => '-q production -R"select[mem>8500] rusage[mem=8500]" -M8500'},
+        'default' => { 'LSF' => '-q production -R"select[mem>1500] rusage[mem=1500]" -M1500',
+                       'SLURM' => '--partition=production --time=10:00:00 --mem=1500M'},
```

Comment:
You could request more memory, on slurm the jobs need a bit more.

4GB is ok

Reply:
Thanks Diana, added more memory.